pycobertura/cobertura.py

```diff
@@ -47,7 +49,15 @@ def __init__(self, report, filesystem=None):
         files referenced in the report. Please check the pycobertura.filesystem
         module in order to discover more about filesystems.
         """
-        self.xml = ET.parse(report).getroot()
+        try:
+            self.xml = ET.fromstring(report).getroot()
+        except lxml.etree.XMLSyntaxError:
+            # it's not a xml-string, so it should be a path.
+            # is it existed?
+            if not os.path.isfile(report):
+                raise FileNotFoundError("file not existed: {}".format(report))
```

Comment:
Can we raise a specific exception? Maybe `InvalidCoverageReport`? We could add this exception under the `Cobertura` class namespace.

```python
class Cobertura(object):
    class InvalidCoverageReport(Exception):
        pass
    ...
```

And then use that to Can we tweak the wording here?

```suggestion
                raise self.InvalidCoverageReport("Invalid coverage report: {}".format(report))
```

Reply:
Ok

Reply:
I noticed that you have added a specific exception in FileSystem.
Maybe we can add a `exceptions` module to manage them like this? 
https://github.com/psf/requests/blob/master/requests/exceptions.py

It looks clear.